enterprise/frontend/src/embedding/data-picker/DataSelector.jsx

```diff
@@ -672,22 +571,13 @@ export class UnconnectedDataSelector extends Component {
     const loadersForSteps = {
       // NOTE: make sure to return the action's resulting promise
       [DATA_BUCKET_STEP]: () => {
-        return Promise.all([
-          this.props.fetchDatabases(this.props.databaseQuery),
```

Comment:
We pass the same query as the line below currently. So we should only have one call here.

https://github.com/metabase/metabase/blob/8fc63096b8d8bfa83d1a4a3878aceac765f0423e/frontend/src/metabase/querying/notebook/components/NotebookDataPicker/EmbeddingDataPicker/EmbeddingDataPicker.tsx#L108